cloudygames/serializers.py

```diff
@@ -3,14 +3,23 @@
 from cloudygames.models import Game, PlayerSaveData, GameSession
 
 class GameSerializer(serializers.HyperlinkedModelSerializer):
+    users = UserSerializer(many=True)
+
     class Meta:
         model = Game
-        fields = ('name', 'publisher', 'max_limit', 'address', 'users')
+        fields = ('id', 'name', 'publisher', 'max_limit', 'address', 'users')
```

Comment:
either include `url` into the fields or use `ModelSerializer` instead of `HyperlinkedModelSerializer`,

otherwise, it defeat the purpose of being `HyperlinkedModelSerializer`